src/features/auth/sign-up/screen/index.tsx

```diff
@@ -1,13 +1,113 @@
-import React from 'react';
-import { Text, View } from 'react-native';
+import React, { useState } from 'react';
+import { Image, ImageBackground, Text, TextInput, TouchableOpacity, View } from 'react-native';
 
+import { AuthStackScreens } from 'navigation/stacks/auth';
+
+import { SignUpRequest } from 'network/models/user-models';
+import { useSignUp } from 'network/queries/user-queries';
+
+import common from '../../../../localization/resources/en.json';
```

Comment:
you can use absolute path, it will required some setups that can be followed with [this article](https://shravanmeena.medium.com/using-absolute-paths-in-react-native-820d6a927a6e) (insted of jsconfig.json 
 is tsconfig.json in your case)

Reply:
I'll take a look at it and add it in the next PR. Thanks